packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/offers/template.html

```diff
@@ -0,0 +1,138 @@
+            data-variant="light"
+            data-on-change="$ctrl.onSelected(modelValue)"
+            data-values="[offer]"
+        >
+            <oui-select-picker-section>
+                <span
+                    class="oui-status oui-status_success"
+                    data-translate="dedicatedCloud_offer_current_offer"
+                    data-ng-if="$ctrl.actualOffer === offer.offerName"
+                ></span>
+                <ul class="offer-list-style">
+                    <li
+                        data-translate="dedicatedCloud_offer_upgrade_custom_report"
+                        data-ng-if="offer.hasCustomReport()"
+                    ></li>
+                    <li
+                        data-translate="dedicatedCloud_offer_upgrade_encrypted_backup"
+                        data-ng-if="offer.hasEncryption()"
+                    ></li>
+                    <li
+                        data-translate="dedicatedCloud_offer_upgrade_retention_points"
+                        data-translate-values="{
+                            retention: offer.getRetentionCount(),
+                        }"
+                    ></li>
+                    <li
+                        data-translate="dedicatedCloud_offer_upgrade_full_backups"
+                        data-translate-values="{
+                            fullBackups: offer.getMinimumFullBackupsCount(),
+                        }"
+                    ></li>
+                    <li
+                        data-translate="dedicatedCloud_offer_upgrade_replication"
+                        data-ng-if="offer.hasReplication()"
+                    ></li>
+                    <li
+                        data-ng-if="offer.hasBackupDays()"
+                        data-translate="dedicatedCloud_offer_upgrade_choice_backup_days"
+                    ></li>
+                    <li
+                        data-ng-if="offer.hasScheduleHour()"
+                        data-translate="dedicatedCloud_offer_upgrade_choice_backup_time"
+                    ></li>
+                    <li
+                        data-ng-if="offer.isPremium()"
+                        data-translate="dedicatedCloud_offer_upgrade_long_term_retention"
+                    ></li>
+                    <li
+                        data-ng-if="offer.hasProxyPerHost()"
+                        data-translate="dedicatedCloud_offer_upgrade_proxy_backup"
+                    ></li>
+                </ul>
+            </oui-select-picker-section>
+            <oui-select-picker-footer>
+                <private-registry-price
+                    data-price="offer.pricings[0].price"
+                    data-tax="offer.pricings[0].tax"
+                    data-user="$ctrl.user"
+                ></private-registry-price>
+            </oui-select-picker-footer>
+        </oui-select-picker>
+    </div>
+    <div data-ng-if="$ctrl.previewMode">
+        <div class="row mb-3">
```

Comment:
`row` can't be children of `row` : https://github.com/twbs/bootlint/wiki/E013
(If compared with L1)